oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -29,6 +29,10 @@ mod wasm;
 /// Nodes must not do any work until the [`Node::start`] method is invoked.
 pub trait Node: Send + Sync {
     /// Starts executing the node.
+    ///
+    /// This method may block, but node implementations should make sure that they create separate
+    /// background threads to execute actual work, and wait on them to terminate when [`Node::stop`]
```

Comment:
I didn't really understand this comment.  Why would `start()` block if the work has to be done in a separate thread?